src/components/events/TournamentBracket.tsx

```
@@ -0,0 +1,93 @@
+import React, { useState } from 'react';
+
+export function TournamentBracket() {
+  // A simple 8-team single elimination bracket mock
+  const [matches, setMatches] = useState([
+    // Quarterfinals
+    { id: 1, round: 1, team1: 'Eagles', team2: 'Tigers', score1: 2, score2: 1, winner: 'Eagles' },
```

ESLint

[error] Replace `'Eagles',·team2:·'Tigers',·score1:·2,·score2:·1,·winner:·'Eagles'` with `"Eagles",·team2:·"Tigers",·score1:·2,·score2:·1,·winner:·"Eagles"`

(prettier/prettier)